installer/seldon-core/description.yaml

```diff
@@ -0,0 +1,23 @@
+name: seldon-core
+product: seldon-core
+version: "1.10.0"
+description: |
+  An MLOps framework to package, deploy, monitor and manage thousands of production machine learning models
+namespace: seldon-system
+install:
+  - type: helm
+    chart: seldon-core-operator
+    repo: https://storage.googleapis.com/seldon-charts
+    values: values.yaml
+    waitfor:
+      - kind: deployment
+        selector: control-plane=seldon-controller-manager
+        namespace: seldon-system
+        condition: available
+        timeout: 600
+uninstall:
+  - type: helm
```

Comment:
just this ? nothing else is required ? not complaining, but I definitely thought you need to duplicate some information from the install step for this to work.

Reply:
I think I don't. I'm using name from the top level and namespace also comes from top level. This should work.